src/NServiceBus.Core/Pipeline/Incoming/PendingTransportOperations.cs

```diff
@@ -0,0 +1,17 @@
+namespace NServiceBus
+{
+    using System.Collections.Generic;
+    using Transports;
+
+    class PendingTransportOperations
+    {  
+        public IEnumerable<TransportOperation> Operations { get { return operations; } }
```

Comment:
How about `IReadOnlyCollection<T>`? I enumerable suggest a potentially infinite sequence for which care must be taken to not enumerate it twice. This guy has an explicit count so looks far more safe for the user.